tide-whisperer.go

```diff
@@ -397,9 +397,9 @@ func main() {
 	go func() {
 		for {
 			sig := <-signals
-			log.Printf(dataAPIPrefix+" Got signal [%s]", sig)
 
 			if sig == syscall.SIGINT || sig == syscall.SIGTERM {
+				log.Printf(dataAPIPrefix+" Got signal [%s]", sig)
```

Comment:
This block can be removed if we only subscribe to the signals we're interested in. 
```
signals := make(chan os.Signal, 1)
signal.Notify(signals, syscall.SIGINT, syscall.SIGTERM)
```